addons/website/static/src/snippets/s_social_media/000.scss

```diff
@@ -82,4 +82,5 @@
             }
         }
     }
+    user-select: none;
```

Comment:
Code style-wise, should this be at the top, right after `$-size: map-get($spacers, 5);` ?